tests/smoke/events.js

```diff
@@ -93,7 +95,7 @@
 				var secondParagraph = editor.editable().getChild(1);
 
 				// no ajax call required on the second run, since words are repeats.
-				observer.assert(["spellCheckComplete", "startMarkTypos", "startScanWords"]);
+				observer.assert(["spellCheckComplete", "startRender", "startScanWords"]);
 				observer.assertRootIs(secondParagraph);
 			}
```

Comment:
Do you think it'd be worth adding a test case just to make sure the new logic for blocking duplicate spellchecks?

I'm thinking a case like

```
<ol><li>something
<ul><li>somethingnested</li></li></ul>
</li></ol>
```

If my understanding is correct, spellchecking "something" will cause "somethingnested" to be skipped